DQM/SiTrackerPhase2/plugins/Phase2TrackerMonitorDigi.cc

```diff
@@ -47,6 +47,7 @@
 Phase2TrackerMonitorDigi::Phase2TrackerMonitorDigi(const edm::ParameterSet& iConfig)
     : config_(iConfig),
       pixelFlag_(config_.getParameter<bool>("PixelPlotFillingFlag")),
+      clsFlag_(config_.getParameter<bool>("OfflineClusteriserFlag")),
```

Comment:
the usage of "Offline" in clusterizer may be misleading here. If I understand correctly this are clusters remade by scratch from the digis (so some sort of home-brew clusters), no?

Reply:
Yes, It is a kind of consecutive digis. Should I change the name of that flag?

Reply:
I would be in favor of that, perhaps `StandAloneClusterizerFlag` or something like that? @sroychow suggestions are welcome.

Reply:
Your suggestion of using StandAloneClusterizerFlag is fine from my side. @gsaha009 could you change the name please?

Reply:
yes, I have changed it.